File: backend/src/main/java/xyz/engsmyre/moviedescriptiongame/exceptions/CustomException.java
package xyz.engsmyre.moviedescriptiongame.exceptions;

import org.springframework.http.HttpStatus;
import org.springframework.web.server.ResponseStatusException;

public class CustomException extends ResponseStatusException {
    public CustomException(HttpStatus status, String reason) {
        super(status, reason);
    }

    @Override
    public Throwable getMostSpecificCause() {
        return null;
    }

    @Override
    public synchronized Throwable fillInStackTrace() {
        return this;
    }
}

File: backend/src/main/java/xyz/engsmyre/moviedescriptiongame/exceptions/CustomException.kt
package xyz.engsmyre.moviedescriptiongame.exceptions

import org.springframework.http.HttpStatus
import org.springframework.web.server.ResponseStatusException

open class CustomException(status: HttpStatus, reason: String?) : ResponseStatusException(status, reason) {
    override fun getMostSpecificCause(): Throwable? {
        return Throwable()
    }

    @Synchronized
    override fun fillInStackTrace(): Throwable {
        return this
    }
}